### PR TITLE
Updating booking validation error codes

### PR DESCRIPTION
**Pull request by tomgag**

### CODE REVIEW

File: api/booking/examples/errors.json [hunk 1]
```diff
@@ -405,6 +405,46 @@
       "message": "Delivery indoor is available in Sweden and Denmark only.",
       "suffixable": false
     },
+    {
+      "code": "BOOK_VALIDATION-008",
+      "message": "Illegal argument.",
```
Comment (AnshikaKoul) [inline api/booking/examples/errors.json]: this is not customs specific? 🤔

**Comment by AnshikaKoul** [comment]
aha so were there more error codes missing from devsite? 👀like not just the ones we introduced in customs?

**Comment by tomgag** [comment]
Yes, and I only vent through BOOK_VALIDATION* Backlog item to autogenerate these?

File: api/booking/examples/errors.json (hunk 1 above)
Comment (tomgag) [reply api/booking/examples/errors.json]: no. used by ApiService

**Pull request merged**